tests/util.rs

```diff
@@ -393,6 +393,18 @@ pub mod omnireplica {
             self.paxos.propose_normal(data).expect("Failed to propose!");
         }
 
+        pub fn garbage_collect(&mut self) {
+            self.paxos.garbage_collect()
+        }
+
+        pub fn garbage_collect_by_index(&mut self, index: u64) {
+            self.paxos.garbage_collect_by_index(index)
+        }
+
+        pub fn get_garbage_collector_index(&self) -> u64 {
+            self.paxos.get_gc_from_storage()
+        }
+
```

Comment:
Do we really need these functions if their body is one line anyway? :D

Reply:
I can delete the first one, but the other two I need for the tests